webwhatsapi/js/wapi.js

```diff
@@ -701,7 +718,7 @@ window.WAPI.getUnreadMessages = function (includeMe, includeNotifications, use_u
         const messages = messageGroupObj.msgs.models;
         for (let i = messages.length - 1; i >= 0; i--) {
             let messageObj = messages[i];
-            if (messageObj.__x_isNewMsg || messageObj.__x_MustSent) {
+            if (messageObj.__x_isNewMsg === true || messageObj.__x_MustSent) {
```

Comment:
you could just use messageObj.isNewMsg here, because otherwise it will be undefined